src/server/api/blockallowlist.ts

```diff
@@ -1,9 +1,22 @@
 import { APIConfig, FastifyTypebox } from '.'
 import { Type } from '@sinclair/typebox'
 
+import createError from 'http-errors'
+
 import Store from '../store/index.js'
 import ActivityPubSystem from '../apsystem.js'
 
+export function validateBlocklistFormat (mention: string): void {
+  const sections = mention.split('@')
```

Comment:
```javascript
const isInstanceBlock = sections[1] === '*' && sections[0] === '' && sections[2] !== '';
```

Reply:
I don't think we need to explicitly handle this case since it's already handled by the other checks.